src/test/java/com/bean00/StatusTest.java

```diff
@@ -0,0 +1,18 @@
+package com.bean00;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StatusTest {
+
+    @Test
+    public void getMessage_getsTheAppropriateMessage_forA404StatusCode() {
```

Comment:
Did you test drive the Status class? I would expect there to be at least two tests if you did.

Reply:
@tbaik, no I didn't fully test drive it. I had `buildMessages` previously implemented as a private method in my `Response` class, so I just refactored it out into the `Status` class. I then added the `getMessage` test to `StatusTest` to make sure it worked. I did consider adding in more tests to fully cover the behavior (the other status codes and messages). However, I remembered that the RFCs mentioned the status messages don't really need to be exact strings.

In hindsight I agree that it would have been helpful to write the test first. It was just that in this case, I already had most of the implementation for the class finished.